ledger-core/appctl/manager.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package appctl
+
+//go:generate minimock -i github.com/insolar/assured-ledger/ledger-core/appctl.Manager -s _mock.go -g
+// Manager provides Ledger's methods related to Pulse.
```

Comment:
Hm. Manage is in appctl, but it provides Ledger's methods related to Pulse.

I'm confused. What is the purpose of this compartment?